### ISSUE TITLE
Problems on Windows 11

### ISSUE THREAD

**Issue by thebaz** [tappi287/simmon_gui#3]
Since I upgraded to Windows 11 SimmonGui doesn't work anymore

**Comment by tappi287** [comment]
I have no way to test this currently. Are you really sure you installed the background service as mentioned in the welcome page?

**Comment by thebaz** [comment]
Yes, I am sure. I will BTW collect the watchman log file and send to you for a better understanding

**Comment by thebaz** [comment]
Unistalled the watchman, cleaned the tasks in Windows, unistalled Simmon, restarted, reinstalled simmon, installed the watchman, created the profile and got it working again!
